Makefile

```diff
@@ -76,11 +79,18 @@ serve:
 oc-clean: oc-down
 	$(PREFIX) rm -rf $(OC_DATA_DIR)
 
-oc-create-all: oc-create-tags oc-create-db oc-create-rabbitmq oc-create-masu
+oc-create-all: oc-create-tags oc-create-configmaps oc-create-db oc-create-rabbitmq oc-create-masu oc-create-worker oc-create-flower
 
 oc-dev-db: oc-create-tags oc-create-db
 
+oc-create-configmaps:
```

Comment:
Let's split out config maps and secrets so they don't have to be deployed together in case you want to change one or the other.